util/entitlements/security/security_linux.go

```diff
@@ -160,3 +127,68 @@ func getFreeLoopID() (int, error) {
 	}
 	return 0, errors.Errorf("error getting free loop device: %v", uerr)
 }
+
+func getAllCaps() ([]string, error) {
+	// allCaps provides a list of capabilities to be granted, some of the capabilities might not be available
+	// if that's the case, capabilities are skipped and warning is printed
+	allCaps := []string{
```

Comment:
There's a limit in here. We shouldn't print warnings for the caps that only exist in the very latest kernels.

Reply:
I tried to keep same capabilities so that jobs don't get _more_ capabilities they have today, but certainly can change to `cap.Known()` which will expand the list.